sphinx_gallery/tests/test_backreferences.py

```diff
@@ -21,7 +21,7 @@
 .. only:: html
 
   .. image:: /fake_dir/images/thumb/sphx_glr_test_file_thumb.png
-    :alt: test title
+    :alt:
```

Comment:
Why not remove the `:alt:` altogether if we are not using it?

Reply:
Strictly speaking, the `alt` is required:

https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt#value

Practically speaking, it makes a big difference for screen readers (AFAIK, but I'm not really a user myself): with `alt=""`, the image is ignored by screen readers, without the `alt` attribute, the screen reader will try to provide *some* information, maybe the file name of the image, which really wouldn't be very helpful.

Reply:
Okay this still seems like a regression -- the `alt` seems important for screen readers.

@alexisthual WDYT about keeping it, but setting the `display: None` for the alt in our CSS? That would remove the visual redundancy on mouseover, but keep screen readers working, right?